src/releasers/ruby-yoshi.ts

```diff
@@ -0,0 +1,137 @@
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { ReleasePR, ReleaseCandidate } from '../release-pr';
+
+import { ConventionalCommits } from '../conventional-commits';
+import { GitHubTag } from '../github';
+import { checkpoint, CheckpointType } from '../util/checkpoint';
+import { Update } from '../updaters/update';
+import { Commit } from '../graphql-to-commits';
+import { CommitSplit } from '../commit-split';
+
+import { Changelog } from '../updaters/changelog';
+import { VersionRB } from '../updaters/version-rb';
+
+export class RubyYoshi extends ReleasePR {
+  protected async _run() {
+    if (!this.lastPackageVersion) {
+      throw Error(
+        'last library version must be provided for ruby-yoshi release type'
+      );
+    }
+    const lastReleaseSha = await this.gh.getTagSha(
+      `${this.packageName}/v${this.lastPackageVersion}`
+    );
+    const commits: Commit[] = await this.commits(lastReleaseSha);
```

Comment:
how do we handle the very first release of a library?

Reply:
If we go with the workflow we talked about earlier, unreleased packages will call release-please with `lastPackageVersion == '0.1.0'`